Grid/grid.html

```diff
@@ -0,0 +1,210 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Grid | Page</title>
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+
+    <link rel="stylesheet" href="../Grid/grid.css">
+    <script src="https://kit.fontawesome.com/c9ffd32afb.js" crossorigin="anonymous"> 
+    </script>
+
+</head>
+<body>
+
+    <nav>
+        <div class="nav-section left-nav-links">
+          <div>
+            <a class="nav-a-link" href="../index.html"> Geeta UI</a>
+           </div>
+        </div>
+       
```

Comment:
- remove redundant spaces in the Html file
- use prettier extension for proper code formatting for your HTML and CSS files
- Rest things are clean and good to go